src/ci/docker/scripts/musl-toolchain.sh

```diff
@@ -33,11 +33,13 @@ shift
 # Apparently applying `-fPIC` everywhere allows them to link successfully.
 export CFLAGS="-fPIC $CFLAGS"
 
-git clone https://github.com/richfelker/musl-cross-make -b v0.9.8
+git clone https://github.com/richfelker/musl-cross-make
```

Comment:
I think we could keep this commented out and mention that the below checkout is temporary so people can bump it easily next time.